NEWS.md

```diff
@@ -1,3 +1,14 @@
+## CHANGES WITH 37:
+
+  * Port sources to the host service infrastructure
+  * ostree.config: add `bootloader` config option
+  * Add a new stage for setting kernel parameters via sysctl.d: `org.osbuild.sysctld`
```

Comment:
Would be nice to always use the same format for e.g. adding new stages, so in this case:
  * `stages/org.osbuild.sysctld`: new stage for setting kernel parameters via sysctl.d

Reply:
Fixed :-)